scripts/clear-test-apps.sh

```diff
@@ -0,0 +1 @@
+npx lerna exec -- npx yalc remove --all
```

Comment:
you can use `yarn` instead of `npx`. `npx` might install a package using `npm` which could cause an issue

Reply:
yarn removes `--` in the command, which makes it cannot be executed. Also, since this command is just for no-hoisted packages, the package manager should not matter that much.